content/en/digital-certificate.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: technology
+---
+
+
+## What it is
+
+A digital certificate — also often referred to as a public key certificate, or SSL certificate — is a digital document used to help secure communications over the network. 
```

Comment:
```suggestion
A (digital) certificate — also often referred to as a public key certificate, or SSL certificate — is a digital document used to help secure communications over the network. 
```

Reply:
I know there was a discussion about whether to call it certificate or digital certificate. How about adding parenthesis to indicate it can be used with or without digital?